config/scannable-common.toml

```diff
@@ -56,12 +56,26 @@
 [ores]
 	#Registry names of blocks considered 'common ores', requiring the common ore scanner module.
 	commonOreBlocks = ["minecraft:clay"]
```

Comment:
Why is clay considered an ore?

Reply:
It is kinda hard to look for in vanilla lol. I can remove em if you want it